src/components/date-picker/DatePicker.vue

```diff
@@ -161,11 +163,11 @@ export default class DatePicker extends Vue {
     this.clearDate()
   }
 
-  @Watch('dateText')
+  @Watch('date')
   @Emit('emitDateSync')
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  private emitDateSync (date: string): string {
-    return this.dateText
+  private emitDateSync (date: Date): string {
+    return this.date
```

Comment:
This is a significant change to what was emitted previously, which would require some rework of all UIs that use this shared component.

I recommend emitting the same string value as before.

Reply:
OK, I'm a bit confused -- what's the difference between emitDate() above and emitDateSync() here -- why do we need both?

Reply:
Good point, I'll emit the string instead of the Date. I'm not too sure what the point of emitDateSync() is. I'll dig through some of the UIs a bit to see if it's being used.

Reply:
It seems it's used in [Edit UI](https://github.com/bcgov/business-edit-ui/blob/main/src/dialogs/ResolutionDateDialog.vue#L22) and a [few places in Create UI](https://github.com/search?q=repo%3Abcgov%2Fbusiness-create-ui+emitDateSync&type=code)

Reply:
So both are needed? Please resolve if no change is needed.